executor/src/test/java/org/k8loud/executor/openstack/DetachVolumeTest.java

```diff
@@ -63,8 +57,8 @@ void testDetachVolumeFailed(Volume.Status status) throws OpenstackException {
         verify(clientV3Mock).useRegion(eq(REGION));
         verify(openstackNovaServiceMock).getServer(eq(SERVER_ID), eq(clientV3Mock));
         verify(openstackCinderService).getVolume(eq(VOLUME_ID), eq(clientV3Mock));
-        verify(volume, times(3)).getStatus();
-        verify(volume, times(2)).getName();
+        verify(volumeMock, times(2)).getStatus();
+        verify(volumeMock, times(1)).getName();
```

Comment:
remove `times(1)`

check for more usage of `times(1)`

Reply:
xd, didn't think twice in this case; just decremented the values